WordPressCom-Stats-iOS/WordPressCom-Stats-iOS-Prefix.pch

```diff
@@ -7,8 +7,7 @@
 #ifdef __OBJC__
 
 #import <Foundation/Foundation.h>
-#define LOG_LEVEL_DEF WordPressComStatsLogLevel
 #import <CocoaLumberjack/CocoaLumberjack.h>
-static const DDLogLevel WordPressComStatsLogLevel = DDLogLevelWarning;
+static const DDLogLevel ddLogLevel = DDLogLevelWarning;
```

Comment:
As discussed on Slack, this won't get properly override by the app, so I did #392